plugin/action/parse_es/parse_es_unit_test.go

```diff
@@ -0,0 +1,147 @@
+package parse_es
```

Comment:
It's a default that test is a unit test. Let's rename another file to `pipeline_test`?

Reply:
`pipeline_test` will intersect with other `pipeline_test` tests, is it ok? 
That's probably my issue - I used to name `package_name_test` - for units, `package_name_integration_test` - for integration.